src/_lib/modules/charts/axis/x-axis.directive.ts

```diff
@@ -13,7 +13,7 @@ import { NwXAxisScale } from './models/XAxisScale';
 })
 export class XAxisDirective extends AxisBase {
 
-    @Input() align: 'top' | 'bottom' = 'bottom';
+    @Input() align: 'top' | 'bottom' | 'center' = 'bottom';
```

Comment:
Small suggestion you can probably move this type along with the types used for the y-axis to its own file call it something like Position and use it here and there instead but its fine the way it is.

Reply:
Thanks Walid. My concern here is that they don't share the same values - only 'center' is common between them. For example, top and bottom would not be valid alignments for the y-axis